package.json

```diff
@@ -15,6 +15,8 @@
     "build:packages": "turbo run build --filter=!docs --filter=!template-*",
     "ci:test": "./scripts/tests-ci.sh",
     "pretest": "turbo run pretest",
+    "depcheck": "knip --dependencies --tags=-knipignore",
+    "knip:fix": "knip --fix",
```

Comment:
It's shouldn't fail, as it's the same command running on CI

Reply:
My bad I misread, that's right because not all of those can be resolved using [`fix`](https://knip.dev/features/auto-fix) which is also still an _experimental_ feature. I included it because it can remove unused dependencies and devDependencies from package.json